src/sql/base/browser/ui/checkbox/checkbox.ts

```diff
@@ -48,6 +48,7 @@ export class Checkbox extends Widget {
 		this.onkeydown(this._el, e => {
 			if (e.equals(KeyCode.Enter)) {
 				this.checked = !this.checked;
+				this._onChange.fire(this.checked);
```

Comment:
checkbox should also handle Space key, while you are on this file, do you mind adding space key here?

Reply:
That's already handled by the base component. If I'm remembering this correctly the reason we do this stuff here is because we want to purposely stop the propagation so that it doesn't go up and get caught by the wizard page (which would do the "OK" action). 

But anyways verified that both enter and space work.